src/components/equations/Endurance.js

```diff
@@ -5,15 +5,26 @@ import eqn from '../../equations/endurance';
 
 
 export class Endurance extends Equation {
-  state = {
-    principal: Profile['netWorth'] || 100000,
-    burnRate:  Profile['burnRate'] || 10
+  constructor(props) {
+    super(props);
+    this.state = {
+      principal: Profile['netWorth'] || 100000,
+      burnRate:  Profile['burnRate'] || 10,
+    }
+    this.eqn = eqn.remaining;
+  }
+  
+  componentDidMount() {
+    this.updateEqn()
   }
-  eqn = eqn.remaining;
 
-  updateEqn(_) {
-    _['endurance'] = eqn.endurance(_.principal, _.burnRate);
-    return _
+  updateEqn() {
+    this.setState({
+      endurance: eqn.endurance(
+        this.state.principal,
+        this.state.burnRate
+      )
+    }, () => console.log(this.state))
   }
```

Comment:
If the Endurance Component continues to exist (which maybe we're saying we don't need it), I think we should consider this `updateEqn(_)` hook which isolates React stuff into `EquationComponent`.

Reply:
Not sure what you mean? Consider it how? I don’t think we’ll need another function beyond what’s in the `Config` class

Reply:
Do these src/components/equations/Enderance[Component].js classes stick around? Or do we get down to one generic <Equation/> Component that takes a class?